EPAS 15 - Update as per DB-2202 (#3882)
EnterpriseDB/docs: product_docs/docs/epas/15/epas_guide/03_database_administration/05_edb_audit_logging/01_audit_logging_configuration_parameters.mdx

`edb_audit_archiver_compress_size_limit`

>  Specifies a file size threshold, in megabytes, after which audit logs are eligible for compression. If the parameter is set to `-1`, compression of the log file on a size basis doesn't occur. The threshold applies to multiple audit logs such that they are eligible for compression if their combined size exceeds the threshold size limit, even though the size of each audit log is smaller.

Reviewer: ```suggestion
Specifies a file size threshold, in megabytes, after which audit logs are eligible for compression. If you have multiple audit logs whose combined size exceeds the threshold, the individual files are eligible for compression even though the size of each individual audit log may not meet the threshold. 
 
If the parameter is set to `-1`, compression of the log file on a size basis doesn't occur. 
```
